src/containers/find-offer/offer-filter-block/offer-filter-list/OfferFilterList.tsx

```diff
@@ -60,12 +60,14 @@ const OfferFilterList: FC<OfferFilterListProps> = ({
   const handleFilterChange = (key: keyof FindOffersFilters) => () =>
     updateFilterByKey(key)
 
+  const getOptionLabelForLanguage = (option: LanguageFilter | null): string => {
```

Comment:
```suggestion
  const getOptionLabelForLanguage = (option: LanguageFilter | null) => {
```
Is a return type here necessary?